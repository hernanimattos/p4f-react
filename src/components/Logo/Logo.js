import React, { Fragment } from 'react';

import './Logo.css';

const Logo = () => {
  return (
    <Fragment>
      <img
        className="logo"
        src="https://apostasesportivas24.com/images/apostasesportivas24/logo_p4f.jpg"
        alt="p4f"
      />
    </Fragment>
  );
};

export default Logo;
