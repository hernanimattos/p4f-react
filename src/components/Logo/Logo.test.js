import React from 'react';
import { render } from '@testing-library/react';
import Logo from './Logo';

test('Snap', () => {
  // const renderCP = render(<Logo />);
  const LogoSnap = Logo.toString();
  expect(LogoSnap).toMatchSnapshot();
});

test('Logo', () => {
  const { container } = render(<Logo />);
  expect(container.firstChild).toHaveClass('logo');
});
