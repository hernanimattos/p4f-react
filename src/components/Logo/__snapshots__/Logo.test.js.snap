// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snap 1`] = `
"() => {
  return /*#__PURE__*/_react.default.createElement(_react.Fragment, {
    __self: void 0,
    __source: {
      fileName: _jsxFileName,
      lineNumber: 7,
      columnNumber: 5
    }
  }, /*#__PURE__*/_react.default.createElement(\\"img\\", {
    className: \\"logo\\",
    src: \\"https://apostasesportivas24.com/images/apostasesportivas24/logo_p4f.jpg\\",
    alt: \\"p4f\\",
    __self: void 0,
    __source: {
      fileName: _jsxFileName,
      lineNumber: 8,
      columnNumber: 7
    }
  }));
}"
`;
